src/ucs/sys/sys.h

```diff
@@ -628,6 +630,19 @@ long ucs_sys_get_num_cpus();
  */
 unsigned long ucs_sys_get_proc_create_time(pid_t pid);
 
+/*
+ * Get the current max locked memory limit.
+ *
+ * @param [out] rlimit_value If successful, set to the current limit value.
+ */
+ucs_status_t ucs_sys_get_memlock_rlimit(rlim_t *rlimit_value);
```

Comment:
IMO it better be size_t instead of rlim_t

Reply:
UCS should be as abstraction layer over OS services
So, ucs_sys_get_memlock_rlimit should return SIZE_MAX in case of RLIM_INFINITY